### DISCUSSION TITLE
Adding selected contacts to existing group(s) in 'Active' folder in 'Contacts' tab

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4969] [Ideas]
User requested being able to select multiple contacts in the 'Active' folder and adding them to an existing group(s) via a button like the 'Start Flow' or 'Archive' buttons on the same page. 

From user: 

When you're in the 'Active' folder in the dashboard, you can select a single contact (or multiple contacts), and it gives the option to either: 1) Archive them, 2) Send a new broadcast to them, 3) Block them, or 4) Start a flow in a flow. There needs to also be a way for selected contacts to be added to an existing group, without having to push them to a background flow to achieve this.

Similar to the 'Send Broadcast' or 'Start Flow' popups that you see whenever you select contacts in your directory, there should be a popup for 'manage group memberships', where you can select the groups you want your selected contacts to be added to.

**Comment by ericnewcomer** [comment]
This sounds... familiar.
